spec/spec_helper.rb

```diff
@@ -48,5 +48,6 @@ def stub_request(*args)
     end
   end)
 
+  config.before(:all) { WorkOS.key ||= '' }
```

Comment:
Usually we don't really need a key.. it just needs to not be `nil`. This is fine enough to run the test suite. If we want an API key this will allow that with

```
WORKOS_API_KEY=<api> rspec
```